### ISSUE TITLE
Feature request: ARM64 dockerfile

### ISSUE THREAD

**Issue by egeu5** [cbarraco/google-assistant-over-rest#1]
You're project seems really interesting to me. I did not find any other solution to execute a command from a google-speaker (e.g. play music from Spotify). 

Is it might possible to generate a arm64 dockerfile to use it on a Raspberry pi?

Thank you!

**Comment by cbarraco** [comment]
Done! An ARM64 image should be available on Docker Hub now.

**Comment by egeu5** [comment]
Wow, thats a quick reply - thank you! I tried id. After setting all those google stuff up the following error msg came up in the docker-log:

> Fatal Python error: init_interp_main: can't initialize time
>Python runtime state: core initialized
>PermissionError: [Errno 1] Operation not permitted

>Current thread 0xb6f8b010 (most recent call first):
>no Python frame>

I bind /etc/localtime to the container without a difference. Any ideas? Probaply this is a way to go: [linuxserver-docker-issue](https://github.com/linuxserver/docker-papermerge/issues/4#issuecomment-735236388)

**Comment by cbarraco** [comment]
Yes, it seems that libseccomp library was causing issues for a bunch of containers, I don't have a way to test if that works but its worth a try. I think another way to go might be to run the docker with --privileged if that's an option in your setup.
